tests/e2e/klickerKnight.test.ts

```diff
@@ -1,15 +1,61 @@
 import { expect } from 'chai';
 import { execSync } from 'child_process';
 import { databaseUtil } from '../../src/utils/databaseUtil';
+import { INVALID_COMMAND } from '../testHelpers/invalidCommand';
 import { testIntegration } from '../testHelpers/semanticMocha';
 
+const run = (command: string): string =>
+  execSync(`npm run --silent klicker-knight ${command}`).toString();
+
+const cleanupSave = () => databaseUtil.delete();
+
 testIntegration('klicker-knight', ({ testScenario }) => {
-  testScenario('always')
-    .annihilate(() => {
-      databaseUtil.delete();
+  testScenario('without a command')
+    .annihilate(cleanupSave)
+    .act(() => run(''))
+    .assert('outputs a room description', (arranged, result) => {
+      expect(result).to.include('You are in example room 1');
+    })
+    .assert('outputs available commands', (arranged, result) => {
+      expect(result).to.include('Available Commands: leave');
+    });
+
+  testScenario('with a valid command')
+    .annihilate(cleanupSave)
+    .act(() => run('leave'))
+    .assert('outputs a command description', (arranged, result) => {
+      expect(result).to.include('You leave example room 1');
+    })
+    .assert('outputs a room description', (arranged, result) => {
+      expect(result).to.include('You are in example room 1');
+    })
+    .assert('outputs available commands', (arranged, result) => {
+      expect(result).to.include('Available Commands: leave');
+    });
+
+  testScenario('with an invalid command')
+    .annihilate(cleanupSave)
+    .act(() => run(INVALID_COMMAND))
+    .assert('outputs a failed command description', (arranged, result) => {
+      expect(result).to.include('You cannot do that');
+    })
+    .assert('outputs a room description', (arranged, result) => {
+      expect(result).to.include('You are in example room 1');
+    })
+    .assert('outputs available commands', (arranged, result) => {
+      expect(result).to.include('Available Commands: leave');
+    });
+
+  testScenario('with a literal "default" command')
```

Comment:
Shouldn't this be valid? Or should we think about it like... The "default command" is different from the command "default". A room could have both things and they could be different.

Reply:
I implemented the default command as a symbol to differentiate between `klicker-knight` and `klicker-knight default`. Imagine you're in the loan shark room and you run out of money:

```
# command would be the symbol DEFAULT_COMMAND
klicker-knight

The nice fellow is waiting for you to pay your loan
Available Commands: pay, default
```

```
# command would be the string 'default'
klicker-knight default

He turns into a shark
Available Commands: run, swim
```